heap/heap.py

```diff
@@ -6,19 +6,46 @@ def __init__(self):
     self.size = 0
 
   def insert(self, value):
-    pass
+    self.storage.append(value)
+    self.size += 1
+    self._bubble_up(self.size)
 
   def delete(self):
-    pass
+    max = self.storage[1] #store max so we can return it
+    #replace first storage el w last el in the heap
+    if len(self.storage) == 1:
```

Comment:
comments need to spell stuff out clearly..thats the point of comments...what is 'el' here...element?